### PR TITLE
RESPA-96: Extend reservation endpoint resource filter to accept multiple values

### PR DESCRIPTION
**Pull request by tuomas777**

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -557,7 +557,7 @@ paths:
           type: integer
         - name: resource
           in: query
-          description: Resource id, for filtering reservations by resource
+          description: Resource id, for filtering reservations by resource. Accepts multiple comma-separated values.
```
Comment (jussih) [inline swagger.yaml]: Does this also work by providing multiples of the same query parameter?
As in `localhost:8000/v1/reservation?resource=abc&resource=123`

**Comment by jussih** [comment]
The CI checks are blocked because the PR is coming from a fork. Looking at the code everything looks good though.

File: swagger.yaml (hunk 1 above)
Comment (tuomas777) [reply swagger.yaml]: Pretty sure it does not, should it? I don't think other filters in Respa do that either (?)

**Comment by tuomas777** [comment]
Travis build is now fixed.

**Pull request merged**